.github/CODEOWNERS

```diff
@@ -1 +1,12 @@
 *   @celonis/navi @gencblakqorip9 @celonis/astro
+
+#/*                              @celonis/astro
+#/src                            @celonis/astro
+#/src/core                       @celonis/astro
+#/src/commands/pacman/           @celonis/astro
+#/src/commands/profile/          @celonis/astro
```

Comment:
The first line `*` should cover these paths as well. 
```suggestion
```

Reply:
The `*` would define the set owners everywhere in the repo. We would not like to have ownership of the other team commands. Similar to cfg-ibc ownership.

Reply:
So if we forget to add codeowners for a new folder structure, we would rather have it orphan and not assign a fallback owner?

Reply:
Do you have another idea in mind?

Reply:
I was thinking that we should leave `* @celonis/astro`, and only specify overrides of ownership for specific folder structures. This would make sure all changes are covered from repository rules and require PRs for respective owners.

Reply:
Astro would get added as an unnecessary reviewer for each command change. I don't think we would want that.

Either way, we will have the `* @celonis/astro` until the refactoring is done, so we can decide then if we want to remove it.